### PR TITLE
added dw conv mapping

### PR DESCRIPTION
**Pull request by SeahK**

### CODE REVIEW

File: src/main/scala/gemmini/LoopConv.scala [hunk 1]
```diff
@@ -1055,7 +1058,7 @@ class LoopConv (block_size: Int, coreMaxAddrBits: Int, rob_size: Int, max_lds: I
 
         loop_being_configured.no_pool := cmd.bits.rs2(0)
         loop_being_configured.downsample := cmd.bits.rs2(1)
-
+		  loop_being_configured.depthwise := cmd.bits.rs2(63)
```
Comment (hngenc) [inline src/main/scala/gemmini/LoopConv.scala:1061]: Can you use spaces for tabbing here?

**Review by hngenc — CHANGES_REQUESTED** [review]
1. Have you double-checked that this doesn't stall on Firesim?
2. Can you also bump gemmini-rocc-tests within this PR?

**Comment by fPecc** [comment]
Hi @SeahK,

Do we have a measure on how much this improves the current depthwise convolution  solution? I mean, if the change you made in gemmini-rocc-tests is also merged.

Thanks!

**Comment by hngenc** [comment]
@fPecc This PR is out-of-date. Fortunately, we already have an improved depthwise conv which was added to Gemmini's `master` branch. We  use it for our hand-written Mobilenet implementation (as shown [here](https://github.com/ucb-bar/gemmini-rocc-tests/blob/e326e7c43457ff08669fe88edcaa395d846474d8/imagenet/mobilenet.c#L120)).

Performance is still pretty poor, but the improvement made it much better. (I don't remember exactly how much faster it made it though).

**Pull request closed without merging**